diff --git a/src/main/java/org/example/RemoveGetDataPointsParamRecipe.java b/src/main/java/org/example/RemoveGetDataPointsParamRecipe.java
index d946387..f306334 100644
--- a/src/main/java/org/example/RemoveGetDataPointsParamRecipe.java
+++ b/src/main/java/org/example/RemoveGetDataPointsParamRecipe.java
@@ -1,4 +1,5 @@ org.openrewrite.java.format.AutoFormat
 package org.example;
+
 import java.util.List;
 import org.openrewrite.ExecutionContext;
 import org.openrewrite.Recipe;

diff --git a/src/main/java/org/example/MissingOverrideAnnotation.java b/src/main/java/org/example/MissingOverrideAnnotation.java
index 20bddc5..6464d13 100644
--- a/src/main/java/org/example/MissingOverrideAnnotation.java
+++ b/src/main/java/org/example/MissingOverrideAnnotation.java
@@ -52,7 +52,7 @@ org.openrewrite.java.format.AutoFormat
             }
         };
     }
-    
+
     public static void main(String[] args) {
 
         String pomXml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +

